models/errors-1.0.1.yaml

```diff
@@ -1,6 +1,10 @@
 Errors:
-  title: Errors
+  required:
+    - errors
+    - trace
+    - type
```

Comment:
Any reason why type is required. Seems to be a breaking change compared to 1.0.0.

Reply:
I guess it could be removed. It's not required per se, but if it's not precent it should default to about/blank - which is quite useless. Should I remove it?

Reply:
More more info: https://www.rfc-editor.org/rfc/rfc9457#name-type